pkg/sip/server.go

```diff
@@ -218,7 +218,19 @@ func (s *Server) Start(agent *sipgo.UserAgent, unhandled sipgo.RequestHandler) e
 	s.sipUnhandled = unhandled
 
 	// Ignore ACKs
-	s.sipSrv.OnAck(func(req *sip.Request, tx sip.ServerTransaction) {})
+	s.sipSrv.OnAck(func(req *sip.Request, tx sip.ServerTransaction) {
+	        fromTag, err := getTagValue(req)
+	        if err != nil {
+	            sipErrorResponse(tx, req)
+	            return
+	        }
+	
+	        if c, found := s.activeCalls[fromTag]; found {
+	            if header, ok := req.To(); ok {
+	                c.to.Params["tag"] = header.Params["tag"]
```

Comment:
We might eventually want to send other SIP commands, so lets store this tag as a new field in the `inboundCall` struct and set it specifically when sending `BYE`.

I will need to check RFC to see if we must use this tag for all future commands or not.